bazel/cc_toolchains/clang_cc_toolchain_config.bzl

```diff
@@ -642,6 +642,19 @@ def _impl(ctx):
         ],
     )
 
+    # A feature that enables poisoning of value stores to detect use after
+    # potential reallocation bugs.
+    poison_value_stores = feature(
+        name = "poison_value_stores",
+        requires = [feature_set(["asan"])],
```

Comment:
This is great, especially that it's clearly catching bugs. Had you considered adding it to `--config=asan` in the .bazelrc, so that it's easier to get as part of asan debugging?

Reply:
It's not ready for that yet, given that it makes all the test fail :) Once the tests pass in this mode, I think it should probably just be enabled whenever ASan is enabled rather than having a separate flag. But we can discuss that later I think.